lib/Sema/PlaygroundTransform.cpp

```diff
@@ -686,6 +694,11 @@ class Instrumenter : InstrumenterBase {
     } else if (auto *MRE = dyn_cast<MemberRefExpr>(*RE)) {
       Expr *B = MRE->getBase();
       ConcreteDeclRef M = MRE->getMember();
+      VarDecl *VD = cast<VarDecl>(M.getDecl());
```

Comment:
I don't think you need to do this, can pass `M.getDecl()` to `shouldLog` directly.

Reply:
Good point, thanks! Removed.